larray/io/array.py

```diff
@@ -611,10 +611,18 @@ def from_string(s, nb_index=None, index_col=None, sep=' ', **kwargs):
 
     Examples
     --------
-    >>> # if one dimension array and default separator ' ', a tabulation must be added in front of the data line
+    >>> # to create a 1D array using the default separator ' ', a nan, NaN or a tabulation character \t 
+    >>> # must be added in front of the data line
     >>> from_string("sex  M  F\\n\\t  0  1")
     sex  M  F
          0  1
+    >>> from_string("sex  M  F\\nnan  0  1")
```

Comment:
Same as above. IMO, it is enough to only mention that you can use nan or NaN instead of \t